client/src/reducers/controls.js

```diff
@@ -323,6 +323,18 @@ const Controls = (
         }
       };
     }
+    case "request user defined gene started": {
+      return {
+        ...state,
+        userDefinedGenesLoading: true
```

Comment:
shouldn't this also be defined in the intial state value (lines 102-134), as we do with other state values?

Reply:
Yup! Nice catch